app/helpers/application_formatters_helper.rb

```diff
@@ -83,6 +83,8 @@ def display_topic(topic, options = {})
   # Format the given datetime
   #
   # @param [DateTime] date The datetime to be formatted
+  # @param [Symbol] format The output format. Use Ruby's defaults or check
```

Comment:
Our aim is to try to prevent developers from jumping around through all the files to find the necessary information.  Pointing to the file location is good, but I think we can go further by providing some examples of the formats - `:long => 2016-08-01 16:08:01`, and `:short => 08 Aug 14:05` (as well as those we added for easy reference).